websocket_message_test.go

```diff
@@ -76,231 +76,435 @@ func TestWebsocketMessage_Alias(t *testing.T) {
 func TestWebsocketMessage_CloseMessage(t *testing.T) {
 	reporter := newMockReporter(t)
 
-	msg := NewWebsocketMessage(reporter, websocket.CloseMessage, nil, 1000)
-
-	msg.CloseMessage()
-	msg.chain.assert(t, success)
-	msg.chain.clear()
-
-	msg.NotCloseMessage()
-	msg.chain.assert(t, failure)
-	msg.chain.clear()
-
-	msg.BinaryMessage()
-	msg.chain.assert(t, failure)
-	msg.chain.clear()
+	msg := NewWebsocketMessage(reporter, websocket.CloseMessage, nil, 10)
```

Comment:
Instead of reusing same message and clearing it's chain, could you rework test to create a fresh reporter and message inside each subtest?

This is a much cleaner approach, because we can be sure that subtests don't affect each other, and we're trying to gradually switch tests to these approach.

Reply:
Same for other tests.

Reply:
done